specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/TransferProduct.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01-preview",
+    "billingAccountId": "{billingAccountId}",

```

Comment:
Please can you replace instances of `{billingAccountId}` and `{invoiceSectionId1}` with strings that resemble actual data that your service would use.